packages/autocomplete-plugin-recent-searches/package.json

```diff
@@ -30,7 +30,12 @@
     "prepare": "yarn run build:esm",
     "watch": "watch \"yarn on:change\" --ignoreDirectoryPattern \"/dist/\""
   },
+  "devDependencies": {
+    "@algolia/autocomplete-core": "^1.0.0-alpha.34",
```

Comment:
do you want to pin these? or is there any reason why this isn't pinned and [this](https://github.com/algolia/autocomplete.js/pull/355/files#diff-bcb3310ff2777a844a689b917f2506bca232ee5f401cf57e8da085d3229ef118R38) gets pinned?

Reply:
Yes I plan to pin this (but I'm working on another PR that fixes these inconsistencies).